anymail/webhooks/sendgrid.py

```diff
@@ -69,10 +69,12 @@ def esp_to_anymail_event(self, esp_event):
         else:
             metadata = {}
 
+        anymail_id = esp_event.get('anymail-id', None)
+        smtp_id = esp_event.get('smtp-id', None)
```

Comment:
This could be simplified to `message_id = esp_event.get('anymail_id', esp_event.get('smtp-id'))`.
(Use 'anymail_id' if present, else use 'smtp-id' for backwards compatibility.)